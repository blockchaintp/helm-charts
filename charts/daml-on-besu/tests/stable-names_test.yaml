suite: daml-on-besu.test for stable names
templates:
  - besu-service.yaml
  - besu-serviceaccount.yaml
  - besu-statefulset.yaml
  - daml-service.yaml
  - daml-statefulset.yaml
  - enode-service.yaml
release:
  name: test
  namespace: testns
tests:
  - it: check for stable entity names (services, statefulsets, serviceaccounts)
    asserts:
      - matchSnapshot:
          path: metadata.name
  - it: check for stable PVC names
    templates:
      - besu-statefulset.yaml
      - daml-statefulset.yaml
    set:
      besu:
        persistence:
          enabled: true
      postgres:
        persistence:
          enabled: true
    asserts:
      - matchSnapshot:
          path: spec.volumeClaimTemplates[0].metadata.name
