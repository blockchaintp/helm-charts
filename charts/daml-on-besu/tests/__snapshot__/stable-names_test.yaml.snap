check for stable PVC names:
  1: |
    data
  2: |
    pgdata
check for stable entity names (services, statefulsets, serviceaccounts):
  1: |
    test-daml-on-besu
  2: |
    test-daml-on-besu
  3: |
    test-daml-on-besu
  4: |
    test-daml-on-besu-rpc
  5: |
    test-daml-on-besu-rpc
  6: |
    test-daml-on-besu-0
  7: |
    test-daml-on-besu-1
  8: |
    test-daml-on-besu-2
  9: |
    test-daml-on-besu-3
